lua/nvim-tree/notify.lua

```diff
@@ -1,7 +1,12 @@
 local M = {}
 
+local fallbackHandler = function(msg, level, opts)
+  vim.notify(string.format("[%s] %s", opts.title, vim.inspect(msg)), level)
+end
+
 local config = {
   threshold = vim.log.levels.INFO,
+  handler = fallbackHandler,
```

Comment:
Unnecessary, set during setup.

Reply:
Indeed, removed.

Reply:
I've now recalled why I did it like that, it was because of that comment https://github.com/nvim-tree/nvim-tree.lua/pull/1693#discussion_r1009294846 

Let me know if I should restore it

Reply:
> I've now recalled why I did it like that, it was because of that comment [#1693 (comment)](https://github.com/nvim-tree/nvim-tree.lua/pull/1693#discussion_r1009294846)
> 
> So the notify module could be called before `setup` function gets called, and in such a case a `fallback_handler` would be used. Let me know if I should restore it.

Ah yes, thank you for the diligence.